R/calc_views.R

```diff
@@ -50,7 +51,9 @@ group_views <- function(gsplot){
       views[[v.i]] <- append(views[[v.i]], to_draw)
       views[[v.i]][['window']][['par']] <- append_replace(views[[v.i]][['window']][['par']], tail.gs[['gs.config']][['par']])
     } else{
-      views <- append(views, list(view = append(to_draw, list(window=list(side=add_sides,par=tail.gs[['gs.config']][['par']])))))
+      new.view <- list(append(to_draw, list(window=list(side=add_sides,par=tail.gs[['gs.config']][['par']])))) %>% 
```

Comment:
no, not yet. Right now the side=c(x,y) is a two-side thing. I'd need more clarification on what we expect on side=c(1,2,3,4) - which become the default sides for axis labels? both y and both x? or 1 and 2?